### PR TITLE
fix/#100/synchronise physics

### PR DESCRIPTION
**Pull request by rzimmerdev**

### CODE REVIEW

**Comment by rzimmerdev** [comment]
Fixes Issue #100

File: src/snaskii.c [hunk 1]
```diff
@@ -100,6 +102,26 @@ void init_game()
...
+
+
+void render(scene_t scene)
+{
+    draw_background((char**) scene);
+    draw_snake(&snake);
+    screen_show();
+    draw_menu(&times);
+}
+
```
Comment (kenzonobre) [inline src/snaskii.c:123]: It would be great to comment what these new functions do.

Comment (kenzonobre) [inline src/snaskii.c:123]: It would be great to comment what these new functions do.

Comment (rzimmerdev) [reply src/snaskii.c]: Done!

**Review by kenzonobre — COMMENTED** [review]
Approved!

**Review by kenzonobre — APPROVED** [review]

**Pull request merged**